test/extended/kubevirt/nodereboot.go

```diff
@@ -0,0 +1,85 @@
+package kubevirt
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	k8sapierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	admissionapi "k8s.io/pod-security-admission/api"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = Describe("[sig-kubevirt] node reboot", func() {
```

Comment:
I think we should move to label instead of text; e.g.
```suggestion
var _ = Describe("node reboot", Label("sig-kubevirt"), func() {
```

Reply:
Makes sense - but I just followed the pattern on the other tests here.

I don't mind moving this to labels in a follow up PR.